pvtrace/scene/renderer.py

```diff
@@ -11,6 +11,7 @@
 from pvtrace.geometry.mesh import Mesh
 from pvtrace.light.ray import Ray
 from pvtrace.light.utils import wavelength_to_rgb, rgb_to_hex_int, wavelength_to_hex_int
+from pvtrace.algorithm.photon_tracer import Event  # Refactor to remove dependency on trace algorithm!
```

Comment:
Move `Event` enum into `pvtrace.algorithm.event`

Reply:
Moved to `pvtrace.light.event` these are general events that happen to light rays.